src/app/modules/training-event/components/table-list/table-list.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-12">
+      <div class="card">
+        <div class="card-body">
+          <div class="loading-shade" *ngIf="isLoadingResults">
+            <mat-spinner></mat-spinner>
+          </div>
+          <div class="table-responsive-lg">
+            <table class="table table-hover text-center">
+              <thead class="text-primary d-flex">
+                <th class="col-2 col-sm-1"></th>
```

Comment:
I think we need to display `name`, `time`, `location`, `num_hours` and `num_participants` here. Besides, an `ID` means nothing to our users who don't know our business logic, so please remove it. And please make sure that the number of `<td>` in `<tbody>` matches the number of `<th>` in `<thead>`.